packages/cedar-amcharts/src/render/render.ts

```diff
@@ -75,8 +76,15 @@ export function fillInSpec(spec: any, definition: any) {
     }
   }
 
+  // Handle Legend in case.
   if (definition.legend) {
-    spec.legend.enabled = definition.legend.enable
+    const legend = definition.legend
+    if (legend.hasOwnProperty('visible')) {
+      spec.legend.enabled = legend.visible
+    }
+    if (legend.position && (legend.position === 'top' || legend.position === 'bottom' || legend.position === 'left' || legend.position === 'right')) {
```

Comment:
why not `const supportedLegendPositions = ['top', 'bottom' ...];` at the top of the module and then `if (supportedLegendPositions.includes(legend.position)) {` here?

Reply:
hadn't thought about that. good point.

Reply:
Issue with this is it's es2016/es2017 based. We'll have to change some TS dependencies to get it happy. Thoughts?

Reply:
https://github.com/Microsoft/TypeScript/issues/11890